source/gent/tier1_hortense.rst

```diff
@@ -617,3 +633,26 @@ Getting help
 
 For questions and problems related to Tier-1 Hortense, please contact the central
 support address for Tier-1 compute: `compute@vscentrum.be <mailto:compute@vscentrum.be>`_.
+
+
+Phase 2
+-------
+
+In May 2023 a second phase was installed, adding 48 more nodes to the ``cpu_rome`` partition,
+20 extra GPU nodes with double the CPU and GPU memory in the new ``gpu_rome_a100_80`` partition and
+384 nodes using the newer AMD Milan CPUs called ``cpu_milan``.
+
+The Lustre based scratch storage was also doubled in volume and throughput to a total of 6PB.
+
+With the new GPU nodes, a renaming of the gpu node partitions occured. Users can mosyt likely still
+still use the same ``gpu_rome_a100`` partition that now includes all GPU nodes (and only select the
+``gpu_rome_a100_40`` or ``gpu_rome_a100_80`` for specific cases, e.g. when running multinode GPU jobs
```

Comment:
@boegel i only now realised that for multinode jobs, this will not be transparent

Reply:
Yeah, for multi-node GPU jobs, they should probably be submitting to a more specific GPU partition.

Or would Slurm only start jobs within the same actual partition anyway?

Reply:
Yes, it will not pick nodes from separate partitions afaik.

Reply:
@itkovian can we set something in the partition definition that multinode jobs won't get started?

Reply:
We use `MaxNodes` for that purpose. 

You can also use multiple partitions for a job and let the users pick: either `gpu_rome_a100_40` or `gpu_rome_a100_80` or you specify both (or use this as default for when the partition has not been set for a GPU job). That's what we currently do on Hydra with a job submit plugin.